CODE_OF_CONDUCT.md

```diff
@@ -1,13 +1,3 @@
-# Community Code of Conduct
+# Overview
```

Comment:
As discussed, let's get rid of it and replace it with the doc title - **Conde of Conduct**.